src/library/scala/math/package.scala

```diff
@@ -58,7 +58,21 @@ package object math {
    *          logarithms.
    */
   def exp(x: Double): Double = java.lang.Math.exp(x)
+
+  /** Returns the natural logarithm of a `double` value.
+   * 
+   * The natural logarithm is the logarithm base Euler's number `e`.
+   *
+   *  @param  x the number to take the natural logarithm of
+   *  @return the value `ln(x)` where ln is the natural lograrithm
```

Comment:
The value log<sub>e</sub>(x) is a little too brief if we don't try to define the natural logarithm in terms of Euler's number (so we aren't defining the two circularly).  Could add ', where _e_ is Euler's number', I suppose.